styles.css

```diff
@@ -230,31 +291,50 @@ article{
 	bottom:0;
 }
 
+.item3 .itemText{
+	position:absolute;
+	bottom:0;
+	left:0px;
+}
 
 .smPort h1{
 	display:none;
 }
 
+.blogs a{
+	font-weight:700;
+}
+
 /*Footer*/
 
 #foot{
-	height:100px;
+	height:50px;
 	border:3px solid #000000;
 	padding:5px;
 	margin-top:20px;
 }
 
-/*Media Queries*/
+#email{
```

Comment:
appears to be in the media query section from comment, but not under the @media screen section

Reply:
Resolved